### ISSUE TITLE
fucked up my world

### ISSUE THREAD

**Issue by BruhKidBruh** [Unixkitty/timecontrol#21]
So i downloaded this mod didn't even use it in my world then i uninstalled the mod so i could install realistic sleeping then i realized sleeping doesn't even change time i uninstalled realistic sleeping and it still didn't work now i have to keep this stupid mod installed in my game just to sleep (just tried fixing it now the sun wont move)

**Comment by Unixkitty** [comment]
No, the mod doesn't make any permanent changes to the world besides setting the vanilla gamerule doDaylightCycle to 'false', which you can always reset yourself

**Comment by BruhKidBruh** [comment]
How do i reset the game rules i just want my survival world to be fixxed

**Comment by thenerdski** [comment]
> How do i reset the game rules i just want my survival world to be fixxed

"gamerule doDaylightCycle True" in chat in game. You may have to open the world to LAN in the pause menu, make the change, then close the world and log back in.

**Comment by icetear** [comment]
didn't work for me. Have the same problem. Bed won't skip time anymore after installing Time Control together with Hourglass/BetterDays.

**Comment by Unixkitty** [comment]
The only thing this mod ever changes is the gamerule. Beds might stop working when combining with other mods that affect sleeping or daylight cycle functionality.
